src/components/Carousel.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react'
+import ReactSimplyCarousel from 'react-simply-carousel'
+import CarouselCard from './CarouselCard'
+import CarouselCardOne from '../assets/carousel-card-one.png'
+import CarouselCardTwo from '../assets/carousel-card-two.png'
+import CarouselCardThree from '../assets/carousel-card-three.png'
+import CarouselCardFour from '../assets/carousel-card-four.png'
+
+const Carousel = () => {
```

Comment:
I have never seen the code needed for doing an image carousel before, I looks quite complicated and extensive, so good job with this (including the carousel cards)!